staging/src/k8s.io/kubectl/pkg/cmd/explain/explain_test.go

```diff
@@ -14,24 +14,32 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package explain
+package explain_test
```

Comment:
What is the reason of this change? this makes `explain` command tests unstandard?

Reply:
im not sure what you mean by unstandard, but using _test package with go makes them white box tests, which IMO are preferable to blackbox test package if we can

Reply:
yeah, it means you can't depend on internal, which is definitely better.

Reply:
I mean by "unstandard" comparing to the other commands unit tests which all of them are whitebox tests. But I agree with you.